### PR TITLE
 Prepare 0.8.0 release

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

**Comment by justinfagnani** [comment]
@azakus could you take a look for release?

also, Chrome is failing to launch on Travis. Not sure what's up, passes on other browsers and on Chrome locally for me: https://travis-ci.org/PolymerLabs/lit-html/builds/328322850?utm_source=email&utm_medium=notification

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
    Unreleased section, uncommenting the header as necessary.
 -->
 
-## Unreleased
+<!-- ## Unreleased -->
+
+## [0.7.0] - 2018-01-12
```
Comment (merlinnot) [inline CHANGELOG.md]: @justinfagnani `0.8.0`?

Comment (justinfagnani) [reply CHANGELOG.md]: fixed, thanks!

**Comment by justinfagnani** [comment]
Ok, I'm giving up on Chrome right now, and I'm going to release anyway.

**Pull request merged**